packages/labs/examples/data.ts

```diff
@@ -0,0 +1,111 @@
+    { title: "Modern Times", year: 1936 },
+    { title: "Raiders of the Lost Ark", year: 1981 },
+    { title: "Rear Window", year: 1954 },
+    { title: "The Pianist", year: 2002 },
+    { title: "The Departed", year: 2006 },
+    { title: "Terminator 2: Judgment Day", year: 1991 },
+    { title: "Back to the Future", year: 1985 },
+    { title: "Whiplash", year: 2014 },
+    { title: "Gladiator", year: 2000 },
+    { title: "Memento", year: 2000 },
+    { title: "The Prestige", year: 2006 },
+    { title: "The Lion King", year: 1994 },
+    { title: "Apocalypse Now", year: 1979 },
+    { title: "Alien", year: 1979 },
+    { title: "Sunset Boulevard", year: 1950 },
+    { title: "Dr. Strangelove or: How I Learned to Stop Worrying and Love the Bomb", year: 1964 },
+    { title: "The Great Dictator", year: 1940 },
+    { title: "Cinema Paradiso", year: 1988 },
+    { title: "The Lives of Others", year: 2006 },
+    { title: "Grave of the Fireflies", year: 1988 },
+    { title: "Paths of Glory", year: 1957 },
+    { title: "Django Unchained", year: 2012 },
+    { title: "The Shining", year: 1980 },
+    { title: "WALL·E", year: 2008 },
+    { title: "American Beauty", year: 1999 },
+    { title: "The Dark Knight Rises", year: 2012 },
+    { title: "Princess Mononoke", year: 1997 },
+    { title: "Aliens", year: 1986 },
+    { title: "Oldboy", year: 2003 },
+    { title: "Once Upon a Time in America", year: 1984 },
+    { title: "Witness for the Prosecution", year: 1957 },
+    { title: "Das Boot", year: 1981 },
+    { title: "Citizen Kane", year: 1941 },
+    { title: "North by Northwest", year: 1959 },
+    { title: "Vertigo", year: 1958 },
+    { title: "Star Wars: Episode VI - Return of the Jedi", year: 1983 },
+    { title: "Reservoir Dogs", year: 1992 },
+    { title: "Braveheart", year: 1995 },
+    { title: "M", year: 1931 },
+    { title: "Requiem for a Dream", year: 2000 },
+    { title: "Amélie", year: 2001 },
+    { title: "A Clockwork Orange", year: 1971 },
+    { title: "Like Stars on Earth", year: 2007 },
+    { title: "Taxi Driver", year: 1976 },
+    { title: "Lawrence of Arabia", year: 1962 },
+    { title: "Double Indemnity", year: 1944 },
+    { title: "Eternal Sunshine of the Spotless Mind", year: 2004 },
+    { title: "Amadeus", year: 1984 },
+    { title: "To Kill a Mockingbird", year: 1962 },
+    { title: "Toy Story 3", year: 2010 },
+    { title: "Logan", year: 2017 },
+    { title: "Full Metal Jacket", year: 1987 },
+    { title: "Dangal", year: 2016 },
+    { title: "The Sting", year: 1973 },
+    { title: "2001: A Space Odyssey", year: 1968 },
+    { title: "Singin' in the Rain", year: 1952 },
+    { title: "Toy Story", year: 1995 },
+    { title: "Bicycle Thieves", year: 1948 },
+    { title: "The Kid", year: 1921 },
+    { title: "Inglourious Basterds", year: 2009 },
+    { title: "Snatch", year: 2000 },
+    { title: "3 Idiots", year: 2009 },
+    { title: "Monty Python and the Holy Grail", year: 1975 },
```

Comment:
Would it make sense to select then `F5` in VS Code to alphabetize these? Would placate my inner demon.